packages/core/drizzle/0000_nice_daimon_hellstrom.sql

```diff
@@ -1,10 +0,0 @@
-CREATE SCHEMA "latitude";
---> statement-breakpoint
-CREATE TABLE IF NOT EXISTS "latitude"."document_hierarchies" (
-	"id" bigserial PRIMARY KEY NOT NULL,
-	"parent_id" bigint,
-	"depth" integer NOT NULL,
-	"child_id" bigint NOT NULL,
-	"created_at" timestamp DEFAULT now() NOT NULL,
-	"updated_at" timestamp DEFAULT now() NOT NULL
-);
```

Comment:
What happened?

Reply:
Why you removed an existing migration?

Reply:
struggles with conflicting migrations 🤷🏼 I don't know how to solve them without a drop and generate but that changes all migrations